custom_connectors/custom_auth/trackvia_connector.rb

```diff
@@ -0,0 +1,769 @@
+    update_record: {
+      description: "Update <span class='provider'>record</span> in " \
+      "<span class='provider'>TrackVia</span>.",
+      help: 'Update a record in TrackVia',
+      config_fields: [
+        {
+          name: 'app_name',
+          label: 'App',
+          type: 'string',
+          control_type: 'select',
+          pick_list: 'apps',
+          optional: false,
+          change_on_blur: true,
+          hint: 'Select a TrackVia application from the list above'
+        },
+        {
+          name: 'view_id',
+          label: 'View',
+          type: 'integer',
+          control_type: 'select',
+          pick_list: 'views',
+          pick_list_params: { app_name: 'app_name' },
+          optional: false,
+          hint: 'Select an available view from the list above.'
+        },
+        {
+          name: 'id',
+          type: 'integer',
+          control_type: 'number',
+          optional: false
+        }
+      ],
+      input_fields: lambda do |object_definitions|
+        { name: 'data',
+          type: :array, of: :object,
+          properties: object_definitions['record'] }
+      end,
+
+      execute: lambda do |_connection, input|
+        put("/openapi/views/#{input['view_id']}" \
+        "/records/#{input['id']}")
+          .payload(data: input['data'])
+          .after_error_response(/.*/) do |_code, body, _header, message|
+            error("#{message} : #{body}")
+          end['data']
+      end,
+
+      output_fields: lambda { |object_definitions|
+        object_definitions['response_record']
+      },
+
+      sample_output: lambda { |_connection, input|
+        call(:get_fields_sample_output, view_id: input['view_id'])
+      }
+    },
+    # DELETE requests
+    delete_record: {
+      description: "Delete <span class='provider'> record</span> in " \
+      "<span class='provider'>TrackVia</span>.",
+      help: 'Delete a record in TrackVia',
+      config_fields: [
+        {
+          name: 'app_name',
+          label: 'App',
+          type: 'string',
+          control_type: 'select',
+          pick_list: 'apps',
+          optional: false,
+          change_on_blur: true,
+          hint: 'Select a TrackVia application from the list above'
+        },
+        {
+          name: 'view_id',
+          label: 'View',
+          type: 'integer',
+          control_type: 'select',
+          pick_list: 'views',
+          pick_list_params: { app_name: 'app_name' },
+          optional: false,
+          hint: 'Select an available view from the list above.'
+        },
+        {
+          name: 'id',
+          type: 'integer',
+          control_type: 'number',
+          optional: false
+        }
+      ],
+
+      execute: lambda do |_connection, input|
+        delete("/openapi/views/#{input['view_id']}" \
+        "/records/#{input['id']}")
+          .after_error_response(/.*/) do |_code, body, _header, message|
+          error("#{message} : #{body}")
```

Comment:
https://github.com/workato/custom_connector_docs/pull/40#discussion_r259254536
Are you sure, do not want to use status message i.e. in downstream apps?

Reply:
I agree. Having a status payload for these methods makes sense.